azure-iot-device/azure/iot/device/aio/patch_documentation.py

```diff
@@ -22,28 +22,12 @@ async def disconnect(self):
     disconnect.__doc__ = IoTHubDeviceClient_.disconnect.__doc__
     setattr(IoTHubDeviceClient_, "disconnect", disconnect)
 
-    async def get_storage_info_for_blob(self, blob_name):
```

Comment:
The changes in this file are a result of fixing that issue where get_storage_blob() and notfiby_blob_upload_status() were maintained in the wrong spot

Reply:
Was this file generated?  I know we had a plan to codegen this at one point, but I don't know what the final decision was.  If this is code-generated, it should be marked as such in header comments, including instructions for re-generating it.  If this file is generated, then I assume you re-generated it to get these results?  If not, you should have.

---
In reply to: [489763278](https://github.com/Azure/azure-iot-sdk-python/pull/668#discussion_r489763278) [](ancestors = 489763278)